opal/mca/reachable/netlink/configure.m4

```diff
@@ -50,12 +51,12 @@ dnl - Set $1 to the CPPFLAGS necessary to compile with libnl
 dnl - Set $2 to the LIBS necessary to link with libnl
 dnl - If $3 is 1, AC_MSG_ERROR (i.e., abort) if neither libnl or
 dnl   libnl3 can be found
-dnl - Set HAVE_LIBNL3 to 1 if libnl3 will be used; 0 if libnl1 will be used
-dnl - AC_SUBST $HAVE_LIBNL3
-dnl - AC_DEFINE HAVE_LIBNL3
+dnl - Set OPAL_HAVE_LIBNL3 to 1 if libnl v3 will be used; 0 if libnl v1 will be used
+dnl - AC_SUBST $OPAL_HAVE_LIBNL3
+dnl - AC_DEFINE OPAL_HAVE_LIBNL3
 dnl
 dnl --------------------------------------------------------
-AC_DEFUN([OPAL_REACHABLE_NETLINK_CHECK_LIBNL3],[
+AC_DEFUN([OPAL_REACHABLE_NETLINK_CHECK_LIBNL_xxx],[
```

Comment:
Should this macro be in `config/opal_check_libnl.m4`?  It seems like we want each component that needs to link against libnl to have a `configure.m4` that makes a trivial macro call to do all the libnl magic (e.g., similar to how `opal/mca/btl/openib` calls `OPAL_CHECK_OPENFABRICS` to do the bulk of its work).